### PR TITLE
Dev improvements

### PR DESCRIPTION
**Pull request by benjaoming**
This doesn't mess with production deployment but is just some stuff that's nice if we're gonna encourage people at Bornhack to hack on the site during the Django workshop :)

@valberg what say ye?

Btw. the most obvious improvement is just that you can run `USE_SQLITE=1 make` after checking out the site... before you'd need to get Postgres installed locally.

### CODE REVIEW

**Comment by benjaoming** [comment]
Actually the map in "Info" also works with SQLite, not sure where Postgis or maybe some JSONField is active? But it doesn't seem relevant if people are just supposed to make some simple stuff :)

**Comment by benjaoming** [comment]
A little extra btw.... I *did* check the migration that I changed, it still works in Postgres :)

File: .env_template [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+TICKET_CATEGORY_ID="304e9b44-0b48-472d-a05b-963e52614a69"
+SECRET_KEY="muchsecret"
+ALLOWED_HOSTS="127.0.0.1"
+EMAIL_HOST="localhost"
+EMAIL_PORT="22"
+EMAIL_HOST_USER="$USER"
+EMAIL_HOST_PASSWORD=""
+EMAIL_USE_TLS=""
+DEFAULT_FROM_EMAIL="bornhax@localhost"
+ARCHIVE_EMAIL=""
```
Comment (valberg) [inline .env_template]: We already have https://github.com/benjaoming/bornhack-website/blob/dev-improvements/bornhack/settings/env.dist ;)

File: README.md [hunk 2]
```diff
@@ -24,12 +23,18 @@ Copy environment file and change settings like DATABASE_URL:
...
 
     (venv) $ ./manage.py migrate --settings=bornhack.settings.development
     (venv) $ ./manage.py createsuperuser --settings=bornhack.settings.development
     (venv) $ ./manage.py runserver --settings=bornhack.settings.development
+
+### Setting up Postgres
+
+Using Postgres is only necessary for purposes og using Postgis for a few maps on the site.
+To use default settings and make commands, create a user `bornhack`, password `bornhack`
+and database `bornhack_dev` to use default setttings.
```
Comment (valberg) [inline README.md]: We don't use PostGIS. We use two Postgres specific fields in `shop/models.py`, namely `JSONField` for some payload stuff coming from external services and `DateTimeFieldRange` for product availability.

Comment (valberg) [inline README.md]: Thus you can run everything fine, but once you try to add a product it would blow up :)

File: .env_template (hunk 1 above)
Comment (benjaoming) [inline .env_template]: oh lol, I was making this PR from work I did like a looong time ago.

Anyways, the env.dist won't work, mine is tested... I'll merge them together :)

File: README.md (hunk 2 above)
Comment (benjaoming) [inline README.md]: yeah, I figured before going to bed yesterday that I should fix it so you wouldn't have to write these comments.. then fell asleep.. gimme a sec..

File: .env_template (hunk 1 above)
Comment (benjaoming) [inline .env_template]: Can't find anything in the old env.dist that's not in the template, so should be fine to delete the .dist file...

**Comment by benjaoming** [comment]
@valberg howsaboutsnows?

scruffy

File: shop/migrations/0016_auto_20160529_1122.py [hunk 3]
```diff
@@ -16,7 +16,7 @@ class Migration(migrations.Migration):
         migrations.AddField(
             model_name='coinifycallback',
             name='headers',
-            field=django.contrib.postgres.fields.jsonb.JSONField(default=''),
+            field=django.contrib.postgres.fields.jsonb.JSONField(default=None),
```
Comment (valberg) [inline shop/migrations/0016_auto_20160529_1122.py]: What is this about? :)

**Pull request merged**